### DISCUSSION TITLE
Looking for the latest pre-dotnet version for mono >4

### DISCUSSION THREAD

**Discussion by mabra** [TechnitiumSoftware/DnsServer#604] [Show and tell]
I am on a old linux host and will try experimenting it.
Is an that old version probably available?
Thanks,
Manfred

**Comment by ShreyasZare** [comment]
Thanks for asking. Mono Framework is not supported by the DNS server. 

Mono Framework is the cross platform implementation for the .NET Framework which is a windows only implementation by Microsoft. .NET Framework is now obsolete.

The DNS server uses .NET (Core) runtime by Microsoft which is the successor for .NET Framework and is cross platform. You need to just install the .NET 7 runtime on your linux host to run the DNS server.

PS: Yes, Microsoft is very good at naming their products.
